tests/eho_by_ops.sql

```diff
@@ -0,0 +1,36 @@
+{{ config(
+    severity="error"
+    , tags=["singular_test"]
+    )
+}}
+
+-- Enriched_history_operations table is dependent on the
+-- history_operations table to load. It is assumed that
+-- any id present in the upstream table should be loaded in
+-- the downstream. If records are not present, alert the team.
+WITH find_missing AS (
+  SELECT op.id,
+    op.batch_run_date,
+    op.batch_id
+  FROM {{ source('crypto_stellar', 'history_operations') }} op
+  LEFT OUTER JOIN {{ ref('enriched_history_operations') }} eho
```

Comment:
Same comment here

I think this should be a ref('stg_history_operations') instead of a source. Otherwise these tests would be hardcoded to just prod right?